### ISSUE TITLE
Using keyword arguments causes parameter names used in calling

### ISSUE THREAD

**Issue by cancan101** [blink1073/oct2py#64]
If I write:
```
octave.monofilt(a, nscale=3, minWaveLength=4, mult=2, sigmaOnf=0.65)
```

The generated M code looks like:
```
load /tmp/tmpw3BX7L.mat "A__"
monofilt(A__, "sigmaOnf", 0.65, "minWaveLength", 4, "mult", 2, "nscale", 3)
```
which is not desired. Ideally if keyword arguments are not supported, an exception should be raised to the user.

The correct Python is to drop the keyword arguments and use positional arguments:
```
octave.monofilt(a, 3, 4, 2, 0.65)
```
leading to:
```
monofilt(A__,  3, 4, 2, 0.65 )
```

**Comment by blink1073** [comment]
@cancan101, sorry for the confusion.  The keyword args are meant to emulate *Octave* keyword args, and the behaviour you saw reflects this fact.  If you can certainly call `octave.monofilt(a, 3, 4, 2, 0.65)` directly.

**Comment by blink1073** [comment]
I updated the [docs](http://blink1073.github.io/oct2py/source/info.html#dynamic-functions) as well.

**Comment by blink1073** [comment]
Fixed in 3.0.
